src/components/Apps/ManyJsonTabs/UserJsonTab.vue

```diff
@@ -33,7 +34,7 @@ export default {
           hasExport: false
         },
         tableConfig: {
-          url: `/api/v1/users/users/?${key}=${value}`,
+          url: `/api/v1/users/users/?${key}=${value}&oid=${org_id}`,
           columns: [
             'name', 'username', 'email', 'groups', 'system_roles',
             'org_roles', 'source', 'is_valid'
```

Comment:
None observed. Code is optimized, free of inconsistencies/corrections needed. Please feel free to ask if you require further analysis/optimization.